src/metaImage.cxx

```diff
@@ -2665,7 +2665,6 @@ MetaImage::ReadROIStream(int *           _indexMin,
     }
     else if ("LIST" == m_ElementDataFileName.substr(0, 4))
     {
-      int     fileImageDim = m_NDims - 1;
```

Comment:
Ah, the previous commit should be merged into this one.

Reply:
Thanks.  I don't want to spend too much time making the patches perfect.